Ethereum.md

```diff
@@ -0,0 +1,57 @@
+# About
+
+This is an analysis of the algorithm used in Ethereum to assign validators to committees. Insights can be used as a reference for the design of validator assignment in NEAR. In addition, Ethereum can provide a benchmark for the work required to prove security properties of an algorithm for validator assignment.
+
+# Validator assignment
+
+There are *N* validators with stake *S* and they need to be assigned to validator committees.
+
+The maximum stake per validator is 32 ETH and any amount staked above that limit is ignored ([ref](https://launchpad.ethereum.org/en/faq)). For activation as validator, a node requires a stake of at least 32 ETH. A validator might be penalized for actions like missing votes (inactivity) and malicious behavior, which reduces its staked amount. Once its stake drops below 16 ETH, it will be removed from the set of validators.
+
+# Ethereum’s algorithm
+
+In Ethereum validators are distributed into committees by a [random shuffle](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#compute_committee). Note that the shuffling is independent of validator stakes, i.e. it is unweighted.
+
+## Security
+
+Ethereum operates under the assumption that at least `2/3` of validators are honest. In this case, a malicious validator controls at most `1/3` of attestations. A malicious validator *M* can control a committee if `2/3` of its seats are assigned to *M*. Hence the analysis of the security of Ethereum’s validator assignment algorithm is based on the following question: If there is a malicious validator *M* holding `1/3` of active validator stake, what is the probability of *M* controlling `2/3` of a committee? In other words, if the system as a whole is still secure (`2/3 <= honest validators && M <= 1/3`), what is the probability of *M* controlling a committee? This event is referred to as a *safety failure* and we call the probability of its occurrence *p_sf*.
```

Comment:
I think the threshold for a committee is 2/3 since then a malicious validator has a supermajority. *For a committee*, anything below 2/3 seems to be acceptable since the system can cope with that - I suppose, though I couldn’t pin that down in the specs or docs yet. However, it seems to me like it is implied by [this](https://web.archive.org/web/20190504131341/https://vitalik.ca/files/Ithaca201807_Sharding.pdf) security analysis.

In contrast to the threshold per committee, the critical threshold of the whole system is 1/3 since that’s when attacks become more powerful. Moreover, a malicious validator with stake >= 1/3 may prevent finalization by simply disappearing ([ref](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/attack-and-defense/#attackers-with-33-stake)). If either of that happens in one of many committees, it should have much less of an impact.

>  in a shard

Data sharding was part of the initial Ethereum 2 design, but has been abandoned later on. Yet, the term “shard” still appears in many resources, which can create confusion. Initially a committee should have been responsible for a shard. After abandoning sharding, committees were kept and in subsequent designs there are multiple committees per slot to organize validators. An overview of that development is available [here](https://eth2book.info/capella/part2/building_blocks/committees/#introduction). I'm wondering if it would make sense to add a `# Terminology` section to `Ethereum.md` to explain that?